x/ccv/provider/types/genesis.go

```diff
@@ -95,6 +95,38 @@ func (gs GenesisState) Validate() error {
 	return nil
 }
 
+func (gs GenesisState) ValidateUnbondingOp(ubdOp UnbondingOp) error {
+	if len(ubdOp.UnbondingConsumerChains) == 0 {
+		return sdkerrors.Wrap(ccv.ErrInvalidGenesis, "unbonding operations cannot have an empty consumer chain list")
+	}
+
+	// Check that the ID is set correctly in the UnbondingOpsIndex
```

Comment:
Checks that every unbonding op has the corresponding UnbondingOpsIndex set correctly.